kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain.sh

```diff
@@ -246,56 +246,42 @@ function createDomainHome {
   if [ -z $imagePath ]; then
```

Comment:
It is best to move the following 11 lines of code into the createFiles function.